### PR TITLE
PAYEE-358 - upgrade FE and add twirl date component

### PR DESCRIPTION
**Pull request by masahmad**
- [ ] Unit tests passing
- [ ] Coverage reached
- [x] Acceptance tests passing
- [ ] Reviewed

### CODE REVIEW

**Review by Jamie22Butler — APPROVED** [review]
LGTM 👍

**Review by vivekhmrc — APPROVED** [review]
LGTM 👍

File: app/views/employments/AddEmploymentStartDateFormView.scala.html [hunk 1]
```diff
@@ -17,41 +17,95 @@
...
+
+@this(mainTemplate: MainTempleteNew, form: FormWithCSRF, govukDateInput : GovukDateInput, govukButton: GovukButton, govukErrorSummary: GovukErrorSummary)
+
 
 @(dateForm: Form[LocalDate], employmentName: String)(implicit request: Request[_], messages: Messages, user: controllers.auth.AuthedUser, templateRenderer: uk.gov.hmrc.renderer.TemplateRenderer, ec: scala.concurrent.ExecutionContext)
 
 
 @header = @{
-    ptaHeaderNoH1(
+    ptaHeader(
```
Comment (sergi-hmrc) [inline app/views/employments/AddEmploymentStartDateFormView.scala.html:39]: do we still need the header?

Comment (masahmad) [reply app/views/employments/AddEmploymentStartDateFormView.scala.html:39]: yes its not displayed anymore as the H1 heading is handled by the legend / label  on the form - however I remembered we will need the header later on  when we implement the print functionality so left it in

Comment (sergi-hmrc) [reply app/views/employments/AddEmploymentStartDateFormView.scala.html:39]: although maybe for a form is not needed

Comment (masahmad) [reply app/views/employments/AddEmploymentStartDateFormView.scala.html:39]: good point  - will remove

File: test/views/html/employments/AddEmploymentStartDateFormViewSpec.scala [hunk 2]
```diff
@@ -98,11 +98,10 @@ class AddEmploymentStartDateFormViewSpec extends TaiViewSpec {
...
         def view: Html = template(formWithErrors, employmentName)
 
-        val errorMessage = doc(view).select(".error-message").text
-        val fieldSetError = doc(view).select("form > div").hasClass("form-group-error")
-
+        val errorMessage = doc(view).select(".govuk-error-message").text
+        val fieldSetError = doc(view).select("form div").hasClass("govuk-form-group--error")
         fieldSetError mustBe true
-        errorMessage mustBe globalErrorMessage
+        errorMessage contains globalErrorMessage
```
Comment (brendanstraughan) [inline test/views/html/employments/AddEmploymentStartDateFormViewSpec.scala:104]: how come this has changed?

Comment (masahmad) [reply test/views/html/employments/AddEmploymentStartDateFormViewSpec.scala:104]: If I recall correctly it had something to do with an additional nested element appearing after implementing the new twirl component which caused the test to fail

**Review by brendanstraughan — APPROVED** [review]

**Review by Jamie22Butler — APPROVED** [review]
LGTM

**Pull request merged**